tests/fsharpqa/Source/CodeGen/EmittedIL/TestFunctions/TestFunction24.il.bsl

```diff
@@ -865,15 +865,19 @@
   .method public static class [mscorlib]System.Tuple`2<char,char> 
           pinString() cil managed
   {
-    // Code size       77 (0x4d)
+    // Code size       93 (0x5d)
     .maxstack  6
     .locals init ([0] string str,
              [1] native int pChar,
              [2] string pinned V_2,
              [3] native int V_3,
              [4] int32 V_4,
              [5] native int V_5,
-             [6] int32 V_6)
+             [6] native int V_6,
```

Comment:
This is debug code, and now contains more locals.  It's the only case where this happens, it does surprise me, it relates to NativePtr.get calls:
```
    let pinString() = 
        let str = "Hello World"
        // The following assignment initializes p by using a string.
        use pChar = fixed str
        NativePtr.get pChar 0,  NativePtr.get pChar 1
```
`NativePtr.get` in debug code is now producing more locals. However this is all cross-assembly inlining, not intra-assembly so I'm not certain of the source of this change.

In any case, I've added a further test for the Release optimized code, which hasn't changed.  So I think this change is acceptable.

Reply:
Why is intra-assembly not producing as many locals?

Reply:
We chatted aboutt this.  In this case, intra-assembly debug code has the locals (e.g. for NativePtr.get in FSharp.Core) .  This is what is included in the inline metadata in FSharp.Core.  The names of these locals then get erased when inlining into user code (because inter-assembly inlining).  However the locals themselves are not removed because the code is not re-optimized.

We believe this is ok - ideally this change wouldn't affect such codegen, but this is benign and the extra locals wil be removed by the JIT.  It only affects intra-assembly-inlining-then-inter-assembly-inlining for debug code.